packages/arb-token-bridge-ui/src/util/wagmi/setup.ts

```diff
@@ -131,7 +132,15 @@ export function getProps(targetChainKey: string | null) {
     //
     // https://github.com/wagmi-dev/references/blob/main/packages/connectors/src/walletConnect.ts#L114
     getChains(sanitizeTargetChainKey(targetChainKey)),
-    [customInfuraProvider(), publicProvider()]
+    [
+      customInfuraProvider(),
+      publicProvider(),
+      jsonRpcProvider({
```

Comment:
Why is this diff required in this PR?

Reply:
it is because otherwise the transfer panel will not render

https://github.com/OffchainLabs/arbitrum-token-bridge/pull/1918#discussion_r1773047821